src/features/DebugSession.ts

```diff
@@ -80,6 +80,12 @@ export class DebugSessionFeature implements IFeature {
                     }
                 }
             }
+            else if (config.script) {
+                // In this case, the user has explicitly defined a script path
+                // so make sure to set the cwd to that path if the cwd wasn't
+                // explicitly set
+                config.cwd = config.cwd || config.script;
```

Comment:
What if the config.script is set to `Invoke-Pester`?  Will we try to set that to the CWD?